source/iterator/iterator.go

```diff
@@ -199,26 +203,26 @@ func (iter *Iterator) switchToCDCIterator(ctx context.Context) error {
 	return nil
 }
 
-// checkIfTableExists checks if table exist.
-func checkIfTableExists(ctx context.Context, tx *sql.Tx, table string) (bool, error) {
-	rows, err := tx.QueryContext(ctx, fmt.Sprintf(queryIfTableExists, table))
+func (iter *Iterator) populateKeyColumns(ctx context.Context) error {
+	if len(iter.keyColumns) != 0 {
+		return nil
+	}
+
+	iter.keyColumns = []string{iter.orderingColumn}
+
+	rows, err := iter.repo.DB.QueryxContext(ctx, querySelectPrimaryKeys, iter.table)
 	if err != nil {
-		return false, fmt.Errorf("request with check if the table already exists: %w", err)
+		return fmt.Errorf("select primary keys: %w", err)
 	}
-	defer rows.Close()
 
+	keyColumn := ""
```

Comment:
why not `var keyColumn string`?

Reply:
The only reason is that this is not the beginning of the method.
Do you insist to change it to the `var` type declaration?